docs/fsharp/using-fsharp-on-azure/table-storage.md

```diff
@@ -146,7 +163,7 @@ To update an entity, retrieve it from the Table service, modify the entity objec
 
 ## Insert-or-replace an entity
 
-Sometimes, you don't know if the entity exists in the table or not. And if it does, the current values stored in it are no longer needed. You can use `InsertOrReplace` to create the entity, or replace it if it exists, regardless of its state.
+Sometimes, you don't whether an entity exists in the table. And if it does, the current values stored in it are no longer needed. You can use `InsertOrReplace` to create the entity, or replace it if it exists, regardless of its state.
```

Comment:
you don't whether -> you don't know whether

Reply:
fixed